notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    text_encoder_model = text_encoder['model']
</pre><p>Could you use the same type of quotes in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>